Src/Campus.Master.API/UI/src/app/modules/auth/registration/registration.component.spec.ts

```diff
@@ -3,33 +3,19 @@ import { RegistrationComponent } from './registration.component';
 describe('RegistrationComponent', () => {
     let component: RegistrationComponent;
 
-    const formGroup = {
+    const formBuilder = {
         group: jest.fn()
     } as any;
 
-    const registrationService = {
-        registerUser: jest.fn()
-    } as any;
-
-    const router = {
-        navigate: jest.fn()
-    } as any;
-
-    const toastr = {
-      error: jest.fn()
-    } as any;
-
-    const localeService = {
-      get: jest.fn()
+    const store = {
+        dispatch: jest.fn(),
+        pipe: jest.fn()
     } as any;
 
     beforeAll(() => {
         component = new RegistrationComponent(
```

Comment:
you can try to use TestBed to mock a component with its dependency

Reply:
Done